comms/lrzsz/Portfile

```diff
@@ -22,6 +22,7 @@ checksums           rmd160  1b1776143afaff8bc7447ce998f9d3a958d51f25 \
                     sha256  c28b36b14bddb014d9e9c97c52459852f97bd405f89113f30bee45ed92728ff1 \
                     size    280938
 
+configure.cflags-append -std=c90
```

Comment:
This isn't how we usually fix these kinds of problems. Usually we want to patch the code so it compiles with later C standards, though downgrading to an older standard may be acceptable if it is prohibitively difficult to patch the code. It may be worth investigating whether other distributions (debian, freebsd, gentoo, etc.) have this software, and if so if they have already developed a patch that we can use.

It may even be a simple matter of adding a compiler flag. In the ticket you mentioned duplicate symbol errors. gcc and clang used to default to "common" but now default to "no-common". This code may have been developed under the assumption that "common" would be used. You can force that to happen by adding `-fcommon` to CFLAGS.

Reply:
Fair point, but this is how far as I can take it as the issue is somewhere in the configure script and detecting the right inlining. I'll add some comments to the ticket, that seems the better place.

Reply:
Actually it was multiple implicit return types in configure that caused the issue. I created a manual patch for configure which feels odd.... but I'm definitely not the right person to update via autoconf.

Reply:
There's nothing wrong with adding a `-std` option if the code is written to an older standard. To match older gcc versions' default though, you would need `-std=gnu89`.

Reply:
Perhaps the use of -std=XXX could be added to the official porting guidelines to help both submitters and reviewers. It's the most logical solution, IMO, to support legacy code. Particularly in light of the recent breaking clang v16 changes.